docs/samples/Microsoft.ML.Samples.GPU/Microsoft.ML.Samples.GPU.csproj

```diff
@@ -46,14 +46,17 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.ML.Onnx.TestModels" Version="$(MicrosoftMLOnnxTestModelsVersion)" />
   </ItemGroup>
+  <!-- Tensorflow GPU packages are not available with matching 2.7.0 version -->
```

Comment:
I had to disable these because there are no linux GPU packages https://www.nuget.org/packages/SciSharp.TensorFlow.Redist-Linux-GPU

I'm a bit confused about the versioning of these packages.  I see that `SciSharp.TensorFlow.Redist-Windows-GPU` and `SciSharp.TensorFlow.Redist-Linux-GPU` don't have consistent versions.  @Oceania2018 is this just because folks haven't asked for consistent versions of these be released at the same time?

Reply:
Let's fix this by adding explicit properties for each package.  I'll give a try.